src/features/counterfactual/ActivateAccountFlow.tsx

```diff
@@ -96,6 +96,10 @@ const ActivateAccountFlow = () => {
 
   const { owners, threshold, safeVersion } = undeployedSafeSetup
 
+  const safeToL2SetupDeployment = getSafeToL2SetupDeployment({ version: safeVersion, network: chain?.chainId })
```

Comment:
Same here. We also gotta use 1.4.1:
```suggestion
  const safeToL2SetupDeployment = getSafeToL2SetupDeployment({ version: "1.4.1", network: chain?.chainId })
```